src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/IndexTests.kt

```diff
@@ -3,9 +3,10 @@ package org.vaccineimpact.api.blackboxTests
 import org.junit.Test
 import org.vaccineimpact.api.blackboxTests.schemas.JSONSchema
 import org.vaccineimpact.api.blackboxTests.helpers.RequestHelper
+import org.vaccineimpact.api.test_helpers.DatabaseTest
 import org.vaccineimpact.api.test_helpers.MontaguTests
 
-class IndexTests : MontaguTests()
+class IndexTests : DatabaseTest()
 {
```

Comment:
Now that the API is using the database on every request, this blackbox test must set up the database properly.